pyvcloud/vcd/org.py

```diff
@@ -546,3 +554,25 @@ def get_roles_query(self, name_filter=None):
             equality_filter=name_filter,
             qfilter=org_filter)
         return query, resource_type
+
+    def change_catalog_owner(self, catalog_name, user_name):
+        """
+        Change the ownership of Catalog to a given user
+        :param catalog_name: Catalog whose ownership needs to be changed
+        :param user_name: New Owner of the Catalog
+        :return: None
+        """ # NOQA
+        if self.resource is None:
+            self.resource = self.client.get_resource(self.href)
+        catalog_resource = self.get_catalog_resource(catalog_name, True)
+        catalog_href = catalog_resource.get('href') + '/owner'
```

Comment:
owner link should be extracted off the catalog resource

Reply:
Done